### PR TITLE
[TPG] feat: Add dev container config

### PR DESCRIPTION
**Pull request by lalabuy948**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Typing Dev Container",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "settings": { 
+        "terminal.integrated.shell.linux": "/bin/zsh",
+        "go.gopath": "/usr/local/go"
```
Comment (hyangah) [inline .devcontainer/devcontainer.json]: given that GOROOT is /usr/local/go (according to Dockerfile), I think this setting is strange.
go.gopath will. be unnecessary (or set to $HOME/go or something)

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.12.0
+
+# Install alp packages
+RUN apk add bash zsh curl git
+
+# Install node
+RUN apk add --update nodejs npm
+
+# Install go
+COPY --from=golang:1.14.4-alpine /usr/local/go/ /usr/local/go/
```
Comment (hyangah) [inline .devcontainer/Dockerfile]: Didn't know this is feasible. 

Why not just based on this container from the beginning?
In line 1, 

FROM golang:1.14.4-alpine

Comment (lalabuy948) [reply .devcontainer/Dockerfile]: Yeap, thanks for this. I didn't see the go container for vscode.
changed to this `FROM mcr.microsoft.com/vscode/devcontainers/go:dev-1.15`

**Comment by lalabuy948** [comment]
@hyangah do you have any thoughts on ports? Im trying to access them when container is up, but no luck

**Pull request merged**